.github/workflows/ansible-tests.yml

```diff
@@ -0,0 +1,118 @@
+name: ansible-tests
+on:
+  push:
+
+jobs:
+  ansible-test-python3:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [debian, rocky8, suse]
+        ansible_version: ["2_10", "4_10"]
+        agent_version: ["6", "7"]
+    container:
+      image: datadog/docker-library:ansible_${{ matrix.os }}_${{ matrix.ansible_version }}
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
In fact we need to use v1 here, otherwise it fails with error like that: https://github.com/ansible-collections/Datadog/actions/runs/6261375741/job/17001343545?pr=12
v4 works on some of the tests, but I prefered to use the same version everywhere